packages/calendar/src/LionCalendar.js

```diff
@@ -374,7 +374,9 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
   }
 
   __disableDatesChanged() {
-    this.__ensureValidCentralDate();
+    if (this.__connectedCallbackDone) {
+      this.__ensureValidCentralDate();
```

Comment:
the side effect was caused by initial value of `disableDates` causing this method call and update of `centralDate` before is it even initialised with `selectedDate` value

actually we just need a guard here to not update `centralDate` before full initialisation has finished